### DISCUSSION TITLE
maldive 2021 don't turn on

### DISCUSSION THREAD

**Discussion by mulroux95** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#369] [General]
Good morning all,
I have a 6 wire maldive. I programmed the ESP8266 and I manage to connect to it. My problem is that when I connect the card to the SPA, then the ESP 8266 lights up and you can connect to it in WIFI. However, the spa display remains off and the spa does not start. when I reconnect the spa without the card, it starts normally. Ideas ?

i use eric pbc

**Comment by visualapproach** [comment]
Good evening, to check hardware go to hwtestinfo.html. Also check that you defined the correct model and pcb as shown in the instructions.

**Reply by mulroux95** [reply]
hello, Thank you for your answer.
I did the test and everything is wrong.
I have a doubt about my welding of 8 channel.
![Hardware test](https://user-images.githubusercontent.com/112511099/187971370-fdea1004-ba6e-49ec-ab2e-b5ec962f3c97.jpg)

**Reply by visualapproach** [reply]
Feel free to post images of your device and your model.h

**Comment by mulroux95** [comment]
![Capture d’écran du 2022-09-01 19-20-25](https://user-images.githubusercontent.com/112511099/187974863-8b69f1cc-7df2-4091-b2c3-0139054d4f4f.png)
![Capture d’écran du 2022-09-01 19-19-43](https://user-images.githubusercontent.com/112511099/187974899-babd62a0-6e08-4f81-b264-736462d55b47.png)

**Reply by visualapproach** [reply]
Good, now switch to pcb_v1 instead in the defines and the hwtest will probably be better

**Reply by mulroux95** [reply]
ok but the eric PCB is the pcb V1 or V2. I thought the version of pcb eric was v2.
I have to reflash the ESP is that it?
sorry for my english

**Reply by visualapproach** [reply]
It's v1. Yes recompile and upload via usb or OTA

**Reply by mulroux95** [reply]
good evening I flashed via ota in version 1 and now it's ok. however, I have to press the buttons several times before it is taken into account by the pump. is this normal?

**Reply by visualapproach** [reply]
What buttons? Web gui buttons usually takes one or two seconds before action. The display buttons should be immediate.

**Reply by mulroux95** [reply]
The web gui buttons.

**Reply by visualapproach** [reply]
Have you tried waiting a bit? Bad Wi-Fi? What's the rssi?

**Reply by mulroux95** [reply]
Rssi between -75 & -77. What  ' s this value?
Thanks you

**Reply by visualapproach** [reply]
No problem, thank you for your kindness! 
RSSI is Wi-Fi strength value and -75 is on the low side (closer to zero is better). See if it works better by putting the router closer. Mine is fine with rssi -60 but it's not an absolute value so you should experiment a little
